docs/setup-and-configuration.rst

```diff
@@ -217,6 +217,56 @@ If you want to store files using object storage, open the ``mwdb.ini`` file and
 
 If you use Docker-based setup, all the configuration can be set using environment variables (e.g. ``MWDB_STORAGE_PROVIDER=s3``).
 
+Oauth authentication
```

Comment:
I would move it to separate page like other new features (e.g. Karton integration).